cli/check/check.go

```diff
@@ -41,6 +41,7 @@ func init() { subcommands.Register(&checkCmd{}, "package query") }
 type checkCmd struct {
 	sources string
 	dryRun  bool
+	db_only bool
```

Comment:
dbOnly for the variable name, db_only for the flag name (https://google.github.io/styleguide/go/guide#mixedcaps)